app/views/offers/index.html.erb

```diff
@@ -14,3 +14,5 @@
     <%end %>
   </div>
 </div>
+
+<%=link_to "Add new offer", new_offer_path, class: "btn btn-secondary"%>
```

Comment:
this will also raise a small conflict later with amanda's push, but we can keep what amanada is pushing and just keep this line too, so should also be easy to solve